Project/Packages/manifest.json

```diff
@@ -1,14 +1,23 @@
 {
   "dependencies": {
+    "com.unity.2d.sprite": "1.0.0",
```

Comment:
looks like you opened this and saved it with an editor > 2018.4

Reply:
Yeah, reverted those.